packages/web/index.js

```diff
@@ -192,5 +182,15 @@ module.exports = (neutrino, opts = {}) => {
       }
 
       config.output.filename('[name].[chunkhash].js');
+    })
```

Comment:
Looks like this was moved without modification, so we should probably reset this file to what is in master.